pkgs/development/coq-modules/autosubst-ocaml/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  mkCoqDerivation,
+  coq,
+  version ? null,
+}:
+
+mkCoqDerivation {
+  pname = "autosubst-ocaml";
+  owner = "uds-psl";
+
+  release."1.1+8.19".sha256 = "sha256-AGbhw/6lg4GpDE6hZBhau9DLW7HVXa0UzGvJfSV8oHE=";
+
+  inherit version;
+  defaultVersion =
+    with lib.versions;
+    lib.switch coq.coq-version [
+      {
+        case = isEq "8.19";
+        out = "1.1+8.19";
+      }
+    ] null;
+
+  propagatedBuildInputs = with coq.ocamlPackages; [
```

Comment:
I don’t think these inputs should be propagated.

Reply:
Thanks! I changed it to `buildInputs`.